storehaus-memcache/src/main/scala/com/twitter/storehaus/memcache/MemcacheStore.scala

```diff
@@ -70,7 +70,6 @@ object MemcacheStore {
       .tcpConnectTimeout(timeout)
       .requestTimeout(timeout)
       .connectTimeout(timeout)
-      .readerIdleTimeout(timeout)
```

Comment:
this seems like a maybe logical change. Are we sure there is nothing to replace it?